BitwardenShared/Core/Platform/Services/AuthenticatorSyncService.swift

```diff
@@ -101,6 +109,13 @@ actor DefaultAuthenticatorSyncService: NSObject, AuthenticatorSyncService {
 
     // MARK: Public Methods
 
+    public func getTemporaryTotpItem() async -> AuthenticatorBridgeItemDataView? {
+        guard await configService.getFeatureFlag(FeatureFlag.enableAuthenticatorSync) else {
+            return nil
+        }
+        return try? await authBridgeItemService.fetchTemporaryItem()
```

Comment:
🎨  If an error is thrown we should report it to the logger/crashlytics so we know there's an error happening in the app.

Reply:
Updated in latest and added this to the test as well. 👍